client/layout/guided-tours/tours/boosted-post-tour.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	LinkQuit,
+	makeTour,
+	Quit,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import {
+	isAbTestInVariant,
+	isEnabled,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const BoostedPostTour = makeTour(
+	<Tour
+		name="boostedPostSurvey"
+		path="/posts/"
+		version="20170127"
+		when={ and(
+			isEnabled( 'boosted-post-survey' ),
+			isDesktop,
+			isAbTestInVariant( 'boostedPostSurvey', 'enabled' )
+		) }
+	>
+		<Step name="init" placement="right">
+			<p>{ translate( 'Interested in increasing your readers?' ) }</p>
```

Comment:
Thanks a lot Ben. I made changes. We feel we need the URL for a couple of reasons. One is we don't know whose taking the survey without at least the site URL. Also one of the challenges we face at scale is if people want to boost content that will be inappropriate on other sites (it happened once in a prior test). So we want to get a sample to see what we are facing. 

It looks like the final page on Polldaddy is hard set. I can't find a way to edit it.